packages/tradershub/src/features/cfd/screens/Jurisdiction/JurisdictionCard/JurisdictionCard.classnames.ts

```diff
@@ -0,0 +1,43 @@
+import { cva, VariantProps } from 'class-variance-authority';
+import { ExcludeAllNull } from '@deriv/quill-design';
+
+export const JurisdictionCardClass = cva(
+    'bg-brand-orange items-center rounded-800 border-sm border-system-light-secondary-background cursor-pointer flex flex-col justify-center w-full lg:w-1/4 relative h-full transition-shadow duration-300 transform-gpu',
+    {
+        variants: {
+            isAdded: {
+                true: 'cursor-not-allowed select-none',
+            },
+            isFlipped: {
+                true: 'rotate-180',
+            },
+            isSelected: {
+                true: 'border-sm border-solid border-system-light-primary-background',
+            },
+        },
+    }
+);
+
+export const JurisdictionCardTagClass = cva(
+    'rounded-200 bg-brand-red-darker text-system-light-primary-background px-500 py-[5px]',
+    {
+        defaultVariants: {
+            displayTextSkinColor: 'default',
+        },
+        variants: {
+            displayTextSkinColor: {
+                'brown-dark': 'bg-brand-brown-dark',
+                default: 'bg-brand-red-dark', // default value
```

Comment:
can remove the comment I think we get it that it's a `default value` 😂

Reply:
Double confirmation 😄

Reply:
😂